pkg/parser/translate.go

```diff
@@ -336,8 +336,7 @@ func fromTCPIngressV1beta1(log logrus.FieldLogger, tcpIngressList []*configurati
 		result.SecretNameToSNIs.addFromIngressV1beta1TLS(tcpIngressToNetworkingTLS(ingressSpec.TLS), ingress.Namespace)
 
 		for i, rule := range ingressSpec.Rules {
-
-			if rule.Port <= 0 {
+			if rule.Port <= 0 || rule.Port >= 65536 {
```

Comment:
DRY: the logic for <0 >= 65536 is duplicated.

We could implement `func ValidatePort(n int) bool`, e.g. in package `util`.

Reply:
:+1: added with 667ff0fde40758498bd5f9927b6d174023b39d61